lib/src/models/profile.dart

```diff
@@ -0,0 +1,7 @@
+// ignore_for_file: constant_identifier_names
+
+enum Language {
```

Comment:
I stuck this in `profile.dart` as I anticipate that's where we'll stick other profile-related models. 

I created a `Language` enumeration since these values have to match the values supported by Platform.

Reply:
there is nothing wrong in putting Enums in their own files...

Reply:
True, but Dart seems to encourage grouping things that logically belong together.

https://dart.dev/guides/language/effective-dart/design#consider-declaring-multiple-classes-in-the-same-library

I may reconsider this in the future if it turns into a mess. 🤣